### PR TITLE
upgrade dependencies

### PR DESCRIPTION
**Pull request by cakasuma**
Upgrade dependencies which has outdated lodash version
* "babel-jest"                                  : "^24.1.0"
-- regular expression denial service from dependency braces (low)
* "jest"                                             : "^24.1.0"
-- regular expression denial service from dependency braces (low)
* "ws"                                              : "^6.1.4"
-- Denial of Service  from package ws (high)

### CODE REVIEW

**Comment by negarn** [comment]
do we need to update this repo's version? i.e. `"version": "2.4.2",` change to `"version": "2.4.3",` so we can publish the new version on npm?

**Comment by cakasuma** [comment]
> do we need to update this repo's version? i.e. `"version": "2.4.2",` change to `"version": "2.4.3",` so we can publish the new version on npm?
yes, and after updating this version, i can submit PR to change version in binary static i guess

File: DocumentUploader.js [hunk 1]
```diff
@@ -1 +1 @@
...)}function u(e){var t=e.completion||{};t.type='normal',delete t.arg,e.completion=t}function g(e){this.tryEntries=[{tryLoc:'root'}],e.forEach(d,this),this.reset(!0)}function h(e){if(e){var t=e[_];if(t)return t.call(e);if('function'==typeof e.next)return e;if(!isNaN(e.length)){var r=-1,o=function t(){for(;++r<e.length;)if(m.call(e,r))return t.value=e[r],t.done=!1,t;return t.value=void 0,t.done=!0,t};return o.next=o}}return{next:y}}function y(){return{value:void 0,done:!0}}var f=Object.prototype,m=f.hasOwnProperty,E='function'==typeof Symbol?Symbol:{},_=E.iterator||'@@iterator',S=E.asyncIterator||'@@asyncIterator',x=E.toStringTag||'@@toStringTag',T='object'==typeof e,b=t.regeneratorRuntime;if(b)return void(T&&(e.exports=b));b=t.regeneratorRuntime=T?e.exports:{},b.wrap=r;var P='suspendedStart',O='suspendedYield',v='executing',R='completed',A={},I={};I[_]=function(){return this};var k=Object.getPrototypeOf,L=k&&k(k(h([])));L&&L!==f&&m.call(L,_)&&(I=L);var F=l.prototype=n.prototype=Object.create(I);a.prototype=F.constructor=l,l.constructor=a,l[x]=a.displayName='GeneratorFunction',b.isGeneratorFunction=function(e){var t='function'==typeof e&&e.constructor;return!!t&&(t===a||'GeneratorFunc
```
Comment (negarn) [inline DocumentUploader.js]: why is this file changed?

Comment (cakasuma) [reply DocumentUploader.js]: i run npm build, but i can't publish it. I will revert it back

Comment (negarn) [reply DocumentUploader.js]: you're not supposed to 😅 ashkan or I will publish it after merging. revert is fine, thanks
